### DISCUSSION TITLE
Confused about beta vs next

### DISCUSSION THREAD

**Discussion by bostrom** [semantic-release/semantic-release#2336] [Release workflow] [answered]
Hello. I'm a bit confused over the difference between the `next` and `beta` distributions. They both trigger a pre-release meant for users to try out before making the actual release. But I keep struggling to decide which one to use when developing new features. Is there any simple heuristics involved, or is it simply a matter of taste?

**Comment by gr2m** [comment, marked as answer]
Technically, the difference is that `beta` releases are exclusively used as preview releases, e.g. `2.0.0-beta.1`, `2.0.0-beta.2`, etc.

When creating a `next` release, it will versions that can be promoted to latest later, e.g. `2.0.0` can be released as `next`, tested, then promoted to `latest`.

`next` releases follow semantic versioning. When you release `2.0.0` and find a bug, you will release `2.0.1` as `next`, etc.

`beta` releases bump the pre-release number, no matter the kind of change. If you find a bug in `2.0.0-beta.1`, you fix it and release `2.0.0-beta.2`. If you relealize you have to change an API, you can release `2.0.0-beta.3`. 

Think of `beta` as pre-releases for early adopters, or release candidates. It signals that breaking changes are still possible. Once you use the `next` release channel, no more breaking changes can occure without a breaking version bump.

When working on a new breaking version, you can start with `beta` releases, then do `next` releases, before doing the `latest` release. But you wouldn't use `next` before `beta`.

Hope that all helps :)

**Reply by bostrom** [reply]
Thanks, that actually helps a lot.

**Reply by chalkygames123** [reply]
I have also been wondering the same question, now I understand!

**Reply by j-mcgregor** [reply]
yeah this is a great explanation
